bootloader/src/pyi_utils.c

```diff
@@ -682,9 +682,10 @@ set_dynamic_library_path(const char* path)
      * that we have bundled.
      */
     orig_path = pyi_getenv(env_var);
-    pyi_setenv(env_var_orig, orig_path);
-    VS("LOADER: %s=%s\n", env_var_orig, orig_path);
-
+    if (orig_path) {
+        pyi_setenv(env_var_orig, orig_path);
+        VS("LOADER: %s=%s\n", env_var_orig, orig_path);
+    }
     /* prepend our path to the original path */
     new_path = pyi_strjoin(path, ":", orig_path);
```

Comment:
Yes, but if one is looking at these few lines, he will stumble about why `orig_path` it tested in line 685 and not in 690. The more IMHO precise code would even be (pseudo-code):
```
if orig_path:
   py-setenv …
   VS(…)
   new_path = pyi_strjoin(…)
else:
   new_path = path
```